buildlib/azure-pipelines-perf.yml

```diff
@@ -11,7 +11,7 @@ variables:
 - name: WorkDir
   value: /hpc/scrap/azure/$(Build.DefinitionName)/$(Build.BuildId)-$(Build.BuildNumber)
 - name: perf_nodes
-  value: vulcan03,vulcan04
+  value: vulcan03:24,vulcan04:24
```

Comment:
Can be moved to perf-common.yml.

Reply:
Can move it directly to Bash though, thanks!